applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -189,9 +190,14 @@ def _ValidateSettings(self, settings):
             "turbulence_model": "None"
         }""")
 
-        settings = self._BackwardsCompatibilityHelper(settings)
-        settings.ValidateAndAssignDefaults(default_settings)
-        return settings
+        default_settings.AddMissingParameters(super(NavierStokesSolverMonolithic, cls).GetDefaultSettings())
+        return default_settings
+
+    def ValidateSettings(self):
+        """Overriding python_solver ValidateSettings to call the BackwardsCompatibilityHelper
+        """
+        self.settings = self._BackwardsCompatibilityHelper(self.settings)
```

Comment:
@philbucher  I overrode ValideSettings in this case in order to call the _BackwardsCompatibilityHelper and maintain the same behaviour as before. Is there another intended way of doing this now?

Reply:
see comment above